features/environment.py

```diff
@@ -0,0 +1,41 @@
+from support.naemon_object_config import NaemonObjectConfig
+from support.naemon_system_config import NaemonSystemConfig
+import tempfile
+import os.path
+import signal
+
+
+def before_all(context):
+    os.environ['LD_LIBRARY_PATH'] = (
+        os.getcwd() + '/' + context.config.userdata['shared_libs_path']
+    )
+
+
+def before_scenario(context, scenario):
+    context.wrkdir = os.getcwd()
+    context.tmpdir = tempfile.mkdtemp()
+    assert os.path.exists(context.tmpdir) is not False, (
+        'Failed to create a temporary directory'
+    )
+    print ('Changing directory to %s from %s' % (
+            context.tmpdir, context.wrkdir
+        )
+    )
+    os.chdir(context.tmpdir)
+    context.naemonsysconfig = NaemonSystemConfig()
+    context.naemonobjconfig = NaemonObjectConfig()
+
+
+def after_scenario(context, scenario):
```

Comment:
Sure, even though I have a feeling it could be useful during development to be able to read logs even after a successful run, I guess we shouldn't clutter peoples systems by default when running tests. Will fix.